ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/AbstractConnectorExecutor.java

```diff
@@ -23,4 +30,15 @@ public void initialize(Connector connector) {
             connectorClientConfig = new ConnectorClientConfig();
         }
     }
+
+    public boolean isConnectorPrivateIpEnabled() {
```

Comment:
can we add this to the [MLFeatureEnabledSetting class](https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/settings/MLFeatureEnabledSetting.java)